app/models/manageiq/providers/autosde/inventory/collector/storage_manager.rb

```diff
@@ -26,6 +25,44 @@ def storage_resources
     end
   end
 
+  def san_addresses
+    @san_addresses ||= @manager.autosde_client.StorageHostApi.storage_hosts_get.map do |addresses|
+      addresses_array = []
+
+      addresses.addresses.each do |address|
+        if address.port_type == "ISCSI"
+          port_type = IscsiAddress
```

Comment:
These should be the String `"IscsiAddress"` not the class name

Reply:
fixed